gcloud-java-examples/src/main/java/com/google/gcloud/examples/StorageExample.java

```diff
@@ -121,7 +104,7 @@ public String params() {
       }
       Blob[] blobs = new Blob[args.length - 1];
       for (int i = 1; i < args.length; i++) {
-        blobs[i - 1] = new Blob(storage, args[0], args[i]);
+        blobs[i - 1] = Blob.load(storage, args[0], args[i]);
```

Comment:
I was planning to make major example updates in a different PR, also to use the static `get`, `update` and `delete` methods But I guess it's fine doing it here, I'm on it!